lib/src/helpers/country_finder.dart

```diff
@@ -74,7 +74,8 @@ class CountryFinder {
 
     _filteredCountries = _allCountries.where((country) {
       final countryName = removeDiacritics(country.name.toLowerCase());
-      return countryName.contains(searchTxt);
+      return countryName.contains(searchTxt) ||
```

Comment:
I see this have sneaked itself in. Not sure why, because it was already merge  into dev.